src/main/java/org/jabref/gui/preferences/ai/AiTabViewModel.java

```diff
@@ -54,13 +54,15 @@ public class AiTabViewModel implements PreferenceTabViewModel {
     private final StringProperty mistralAiChatModel = new SimpleStringProperty();
     private final StringProperty geminiChatModel = new SimpleStringProperty();
     private final StringProperty huggingFaceChatModel = new SimpleStringProperty();
+    private final StringProperty chat4AllChatModel = new SimpleStringProperty();
```

Comment:
So many properties...

That's my fault, I designed this code, I plan to fix this, but you done everything correctly!